workflows/pangenome-generate/pangenome-generate.cwl

```diff
@@ -66,3 +97,29 @@ steps:
       originalLabels: relabel/originalLabels
     out: [merged]
     run: merge-metadata.cwl
+  bin_paths:
+    run: ../tools/odgi/odgi_bin.cwl
+    in:
+      sparse_graph_index: buildGraph/odgiGraph
+      bin_width: bin_widths
+    scatter: bin_width
+    out: [ bins, pangenome_sequence ]
```

Comment:
@tetron , these scatter steps are really fast, so I guess we could add a `arv:RunInSingleContainer` hint?